content/articles/2018-02-forcats-0-3-0.Rmd

```diff
@@ -0,0 +1,48 @@
+
+You can install the latest version with:
+
+```{r, eval = FALSE}
+install.packages("forcats")
+```
+
+### Improvements to NA handling
+
+Closing out one of the most up-voted [issues](https://github.com/tidyverse/forcats/issues/89) in the tidyverse, `as_factor()` and `fct_inorder()` can accept NA levels. 
+
+`fct_lump()` now correctly accounts for NA in input. `lvls_revalue()` preserves NA levels. Missing values, including NAs, can be given an explicit factor level using `fct_explicit_na()`, which ensures that missing values will appear in summaries and plots.
+
+### New features
+
+Consistent with other tidyverse packages, all functions that take `...` now use tidy dots. This means that you can use `!!!` to splice in a list of values, and trailing empty arguments are automatically removed. Additionally, all other arguments gain a `.` prefix in order to avoid unhelpful matching of named arguments.
+
+`fct_lump()` has a new argument, `w`, for weighing value frequencies before lumping them together.
+
+### API changes
+
+* `fct_c()` now requires explicit splicing with `!!!` if you have a list of factors that you want to combine.
+
+* `fct_reorder()` and `fct_reorder2()` have renamed `fun` to `.fun` to avoid spurious matching of named arguments.
+
+### Contributors
+
+A big thanks goes to the 35 people who made this release possible by opening issues and contributing code: [\@alistaire47](https://github.com/alistaire47), [\@antaldaniel](https://github.com/antaldaniel), [\@arhot](https://github.com/arhot), [\@BarkleyBG](https://github.com/BarkleyBG), [\@batpigandme](https://github.com/batpigandme), [\@camdenhu](https://github.com/camdenhu), [\@ColinFay](https://github.com/ColinFay), [\@dannyparsons](https://github.com/dannyparsons), [\@davechilders](https://github.com/davechilders), [\@dpavlic](https://github.com/dpavlic), [\@eclarke](https://github.com/eclarke), [\@econandrew](https://github.com/econandrew), [\@felipegerard](https://github.com/felipegerard), [\@geneanalyst](https://github.com/geneanalyst), [\@GlynisHumphrey](https://github.com/GlynisHumphrey), [\@gregrs-uk](https://github.com/gregrs-uk), [\@hadley](https://github.com/hadley), [\@huftis](https://github.com/huftis), [\@instantkaffee](https://github.com/instantkaffee), [\@jmpsteen](https://github.com/jmpsteen), [\@jtr13](https://github.com/jtr13), [\@krlmlr](https://github.com/krlmlr), [\@michaellevy](https://github.com/michaellevy), [\@NBRAYKO](https://github.com/NBRAYKO), [\@pbiecek](https://github.com/pbiecek), [\@philstraforelli](https://github.com/philstraforelli), [\@robinsones](https://github.com/robinsones), [\@rtaph](https://github.com/rtaph), [\@s-fleck](https://github.com/s-fleck), [\@Tazinho](https://github.com/Tazinho), [\@thercast](https://github.com/thercast), [\@vspinu](https://github.com/vspinu), [\@whatevergeek](https://github.com/whatevergeek), [\@wilkox](https://github.com/wilkox), and [\@wkostelecki](https://github.com/wkostelecki).
```

Comment:
Probably can remove yourself from this list 😉

Reply:
Fine— then you're coming out too!